test/bigquery.test.ts

```diff
@@ -301,4 +250,681 @@ describe('BigQueryFormatter', () => {
       `);
     });
   });
+
+  describe('BigQuery DDL Create Statements', () => {
+    const createSchemaVariants = ['CREATE SCHEMA', 'CREATE SCHEMA IF NOT EXISTS'];
+    createSchemaVariants.forEach(createSchema => {
+      it(`Supports ${createSchema}`, () => {
+        const input = `
+          ${createSchema} mydataset
+            DEFAULT COLLATE 'und:ci'
+            OPTIONS(
+              location="us", labels=[("label1","value1"),("label2","value2")])`;
+        const expected = dedent`
+          ${createSchema}
+            mydataset
+          DEFAULT COLLATE
+            'und:ci' OPTIONS(
+              location = "us",
+              labels = [("label1", "value1"), ("label2", "value2")]
+            )
+        `;
+        expect(format(input, { keywordCase: 'upper' })).toBe(expected);
+      });
+    });
+
+    const createTableVariations = [
+      'CREATE TABLE',
+      'CREATE TABLE IF NOT EXISTS',
+      'CREATE OR REPLACE TABLE',
+    ];
+    const createTempTableVariations = [
+      'CREATE TEMP TABLE',
+      'CREATE TEMP TABLE IF NOT EXISTS',
+      'CREATE TEMPORARY TABLE',
+      'CREATE TEMPORARY TABLE IF NOT EXISTS',
+      'CREATE OR REPLACE TEMP TABLE',
+      'CREATE OR REPLACE TEMPORARY TABLE',
+    ];
+
+    createTableVariations.concat(createTempTableVariations).forEach((createTable: string) => {
+      it(`Supports ${createTable}`, () => {
+        const input = `
+          ${createTable} mydataset.newtable
+          (
+            x INT64 OPTIONS(description="desc1"),
+            y STRUCT<
+              a ARRAY<STRING> OPTIONS(description="desc2"),
+              b BOOL
+            >
+          )
+          PARTITION BY _PARTITIONDATE
+          OPTIONS(
+            expiration_timestamp=TIMESTAMP "2025-01-01 00:00:00 UTC",
+            partition_expiration_days=1
+          )`;
+
+        // need to fix the formatting of y STRUCT<a ARRAY<STRING>OPTIONS(description ="desc2"), b BOOL>
```

Comment:
This PR https://github.com/sql-formatter-org/sql-formatter/pull/281 had fixed STRUCT formatting before, but there still are cases

Reply:
Not an issue. Updating the tests.